### ISSUE TITLE
Queue of pending tasks are not deleted

### ISSUE THREAD

**Issue by seredenko** [vyuldashev/laravel-queue-rabbitmq#176]
I have a queue for some tasks - `notifications`.
Some of this tasks need to be delayed, I call method delay() and rabbit make me a queue like `enqueue.notifications.notifications.77467000.x.delay`.
When time to work come, my task migrates from this queue to common `notifications` and done. But delayed queue `enqueue.notifications.notifications.77467000.x.delay` didn't delete.
Can anybody help me?

**Comment by seredenko** [comment]
Or how can I add `x-expires` argument for delayed queue?

**Comment by makasim** [comment]
@seredenko Copy-paste the [RabbitMqDlxDelayStrategy](https://github.com/php-enqueue/enqueue-dev/blob/master/pkg/amqp-tools/RabbitMqDlxDelayStrategy.php) strategy, modify it to meet your needs, set the strategy to AMQP Context.

**Comment by seredenko** [comment]
Yes, it works. But I don't understand how set the strategy to AMQP Context in my laravel project? Fork this lib and change code or there is some more pretty solution?

**Comment by CmdrSharp** [comment]
@seredenko Is this really an issue? If it was delayed, chances are it'll be delayed with the same timeout in the future, meaning the dynamically defined queue can be reused.
